RELEASE_NOTES.md

```diff
@@ -1,7 +1,11 @@
 # Release Notes
 
-## stable-1.15-1.5.0, stable-1.16-1.5.0
+## stable-1.15-1.6.0, stable-1.16-1.6.0
+
+* \[cert-manager\] `usages` is no longer definable as part of `issuerRef`, instead it is a key on its own ([#196](https://github.com/mesosphere/kubernetes-base-addons/pull/196), [@sebbrandt87](https://github.com/sebbrandt87))
```

Comment:
Is this change the reason for the minor bump instead of a patch?

Reply:
Each bi-monthly release is a minor bump. Patch releases are done for patches out of the release cadence, to patch a bug. We did that for 1.4 where it broke the velero backup.

Reply:
Hm there should be a mention of the easy token refresh in these notes https://jira.d2iq.com/browse/D2IQ-65269?src=confmacro

Reply:
Not according to the PR: https://github.com/mesosphere/kubernetes-base-addons/pull/193

Reply:
Ah probably oversight. @jr0d this is a new feature worth calling out, yes? This is for the kubectl plugin with dka instructions page

Reply:
also dka, dex, and opsportal addons all got updated ... maybe we should include something in the release notes about those (unless all related to security plugin)